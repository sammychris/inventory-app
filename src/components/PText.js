import styled from 'styled-components';

const PText = ({children, align}) => {
    return (
        <Container align={align}>
            <P>
                {children}
            </P>
        </Container>
    );
  }


export default PText;

const P = styled.p`
  font-size: 14px;
  color: #383838;
`;

const Container = styled.div` 
  text-align: ${(props) => props.align};
`;