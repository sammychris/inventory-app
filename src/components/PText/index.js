import PText from "./PText";

export default PText;