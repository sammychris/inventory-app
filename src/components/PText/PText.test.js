import { shallow } from "enzyme";
import PText from "./PText";

const wrapper = shallow(<PText />);

test("expect to render a PText", () => {
  expect(wrapper.length).toEqual(1);
});

test("expect to make a snap shot to PText component", () => {
  expect(wrapper).toMatchSnapshot();
});