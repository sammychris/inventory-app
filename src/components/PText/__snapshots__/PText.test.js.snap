// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to make a snap shot to PText component 1`] = `ShallowWrapper {}`;
